webapp/components/sidebar_right_menu.jsx

```diff
@@ -389,6 +389,7 @@ export default class SidebarRightMenu extends React.Component {
                                 />
                             </a>
                         </li>
+                        <li className='divider'></li>
```

Comment:
Please follow the style guide and have this a self closing: `<li className='devider'/>`
This is going to be enforced by my upcoming ESLint changes.